cellbase-app/app/azure/arm/azuredeploy.json

```diff
@@ -0,0 +1,370 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": ""
+            }
+        },
+        "_artifactsLocationSasToken": {
+            "type": "string",
+            "metadata": {
+                "description": ""
+            }
+        },
+        "rgPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "The resource group prefix"
+            }
+        },
+        "rgLocation": {
+            "type": "string",
+            "defaultValue": "uksouth",
+            "metadata": {
+                "description": "Location for deployment"
+            }
+        },
+        "sslCertificateEmail": {
+            "type": "string",
+            "metadata": {
+                "description": "Email address used for mongoDB letsencrypt certificates"
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "defaultValue": "opencgaadmin",
+            "metadata": {
+                "description": "Admin username used by various services"
+            }
+        },
+        "sshAdminKeyData": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The RSA data for the SSH Public Key used to authenticate the following machines: WebServers, Avere, DaemonVM, Mongo, Solr and Zookeeper."
+            }
+        },
+        "mongoDBPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "This Data is used to derive unique passwords for several different services. Select a unique and random combination of data, the longer the better, and do not disclose it."
+            }
+        },
+        "mongoDbDumpUrl": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The URL pointing to a tar.gz zip of the a mongo dump"
+            }
+        },
+        "mongNodeCount": {
+            "type": "int",
+            "defaultValue": 3,
+            "allowedValues": [
+                1,
+                3,
+                5,
+                7,
+                9,
+                11
+            ],
+            "metadata": {
+                "description": "Amount of VMs to deploy for the mongodb replica set"
+            }
+        },
+        "mongoNodeSize": {
+            "type": "string",
+            "defaultValue": "Standard_D4s_v3",
+            "metadata": {
+                "description": "The VM size for the mongo nodes"
+            }
+        },
+        "cellbaseContainerImage":{
+            "type": "string",
+            "defaultValue":  "marrobi/cellbase-app"

```

Comment:
Probably best to pin this to a `tag` and remove the default before we merge (unless an opencga one is now setup). Otherwise people might end up using yours accidentally.

Reply:
done now is on docker hub.